blocksuite/affine/data-view/src/core/view-manager/property.ts

```
@@ -58,125 +70,195 @@ export abstract class PropertyBase<
+  stringValueGet(rowId: string): string | undefined {
+    return this.cellGetOrCreate(rowId).stringValue$.value;
   }
 
   valueGet(rowId: string): RawValue | undefined {
-    return this.cellGet(rowId).value$.value;
+    return this.cellGetOrCreate(rowId).value$.value;
   }
 
   valueSet(rowId: string, value: RawValue | undefined): void {
-    return this.cellGet(rowId).valueSet(value);
+    return this.cellGetOrCreate(rowId).valueSet(value);
```

Biome (1.9.4)

[error] The function should not return a value because its return type is void.

The function is here:

'void' signals the absence of value. The returned value is likely to be ignored by the caller.

(lint/correctness/noVoidTypeReturn)